docs/releases.md

```diff
@@ -21,3 +58,10 @@ https://github.com/mozilla/testpilot/blob/master/docs/development/hosting.md
 This repository is in the "testpilot-mozillaextension" Jenkins pipeline.
 The CloudOps team manages access to, and can help report on, the status of the
 builds.
+
+The resulting files deployed are:
+
+- Updates file for automatic browser extension updates: https://testpilot.firefox.com/files/lockbox@mozilla.com/updates.json
+- Latest version of the signed extension XPI: https://testpilot.firefox.com/files/lockbox@mozilla.com/latest
+
+**IMPORTANT** Join the IRC channel **#testpilot-bots** for updates on the status of these builds prior to pushing the `production` branch to GitHub.
```

Comment:
build/sign/deploy status is reported to IRC *after* pushing to `production` (changes to that branch are what trigger the Jenkins build process)

Reply:
Yes, the status reported after pushing ... but that's starts the race: am I in or did the build finish?

Reply:
If you're in the IRC channel before the pushing, and stay until you see a report, you're guaranteed (as much as IRC guarantees anything) to see that build status.

Reply:
so, is this better?

> **IMPORTANT** Status updates on test-pilot builds are reported in the IRC channel **#testpilot-bots**.  Be sure to join the channel before pushing the `production` branch to GitHub in order to receive the status updates.